cmd/create/server.go

```diff
@@ -72,25 +71,14 @@ func enrollServer(ctx context.Context) {
 func init() {
 	serverEnrollFlags = &serverEnrollParams{}
 
-	serverEnroll.PersistentFlags().StringVar(&serverEnrollFlags.serverID, "server-id", "", "server id to be created. New id will be created if null.")
-	serverEnroll.PersistentFlags().StringVar(&serverEnrollFlags.facility, "facility", "", "facility of the server")
-	serverEnroll.PersistentFlags().StringVar(&serverEnrollFlags.ip, "ip", "", "ip of the server")
-	serverEnroll.PersistentFlags().StringVar(&serverEnrollFlags.username, "user", "", "username of the server")
-	serverEnroll.PersistentFlags().StringVar(&serverEnrollFlags.password, "pwd", "", "password of the server")
-
-	if err := serverEnroll.MarkPersistentFlagRequired("facility"); err != nil {
-		log.Fatal(err)
-	}
-
-	if err := serverEnroll.MarkPersistentFlagRequired("ip"); err != nil {
-		log.Fatal(err)
-	}
-
-	if err := serverEnroll.MarkPersistentFlagRequired("user"); err != nil {
-		log.Fatal(err)
-	}
+	mctl.AddAddressFlag(serverEnroll, &serverEnrollFlags.ip)
```

Comment:
Can we rename this to `mctl.AddBMCAddressFlag` and the flag should be `--bmc-addr`